narwhals/expr.py

```diff
@@ -433,6 +433,43 @@ def std(self, *, ddof: int = 1) -> Self:
         """
         return self.__class__(lambda plx: self._call(plx).std(ddof=ddof))
 
+    def skew(self) -> Self:
+        """
+        Calculate the sample skewness of a column.
+
+        Returns:
+            An expression representing the sample skewness of the column.
+
+        Examples:
+            >>> import pandas as pd
+            >>> import polars as pl
+            >>> import narwhals as nw
+            >>> df_pd = pd.DataFrame({"a": [1, 2, 3, 4, 5], "b": [1, 1, 2, 10, 100]})
+            >>> df_pl = pl.DataFrame({"a": [1, 2, 3, 4, 5], "b": [1, 1, 2, 10, 100]})
+
+            Let's define a dataframe-agnostic function:
+
+            >>> @nw.narwhalify
+            ... def func(df):
+            ...     return df.select(nw.col("a", "b").skew())
+
+            We can then pass either pandas or Polars to `func`:
```

Comment:
If possible I would add an example with pyarrow as well (we trying to do so since #1000)

Reply:
Good one! Added a pyarrow example